content/security-policies.md

```diff
@@ -4,14 +4,15 @@ date: "2020-10-01"
 category: "develop"
 ---
 
-It is essential to consider the security features and various configuration options of your orchestrator to control the security privileges the container runtime can use to spawn containers. The use of higher level policy and governance constructs may enforce those security guardrails).
+It is essential to consider the security features and various configuration options of your orchestrator to control the security privileges the container runtime can use to spawn containers. The use of higher level policy and governance constructs may enforce those security guardrails.
 
 
 ## Projects
 - [OPA](https://www.openpolicyagent.org/)
 - [Kubernetes Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/)
 - [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
-- [Kubernetes Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) 
+- [Kubernetes Pod Security Policy (PSP) - (deprecated, removed in 1.25)](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) 
+- [Kubernetes PSP Replacement Policy (alpha in 1.22)](https://github.com/kubernetes/enhancements/issues/2579)
```

Comment:
Nit: We probably don't need to include this right now. Once there is an implementation we can add this in. WDYT ?

Reply:
I agree that it is too early for PSP replacement. Perhaps the reference to the deprecation of PSP would be useful, as the target group of readers may not know the stage of the lifecycle. But I have removed it with a new commit as it is clearly visible when you go to the link for PSP.